integrations/standalone/tests/pageobjects/TextArea.ts

```diff
@@ -3,8 +3,12 @@ import { Locator, expect } from '@playwright/test';
 export class TextArea {
   private readonly locator: Locator;
 
-  constructor(parentLocator: Locator, label: string) {
-    this.locator = parentLocator.getByLabel(label);
+  constructor(parentLocator: Locator, label: string, isJavaClassSelector?: boolean) {

```

Comment:
Don't understand this change...
This is a generic TextArea page object for the generic TextArea widget and this shouldn't know anything of the JavaClassSelector...

You should anyway use the Combobox pageobject for your tests as the java class is a combobox...